node/windows-packaging/install-calico-windows.ps1

```diff
@@ -21,8 +21,9 @@
 Param(
     # Note: This URL only works for releases. To test development code, build calico-windows.zip from source.
     # VERSION is replaced by our build tooling.
-    [parameter(Mandatory = $false)] $ReleaseBaseURL="https://github.com/projectcalico/calico/releases/download/VERSION/",
-    [parameter(Mandatory = $false)] $ReleaseFile="calico-windows-VERSION.zip",
+    [parameter(Mandatory = $false)] $CalicoVersion="v3.25.0",
```

Comment:
What's the rationale we're using for replacing `VERSION` with `$CalicoVersion`? Is it just that we can support hardcoding `$CalicoVersion` separately?

I think if we decide to go this route, we could do something like https://github.com/projectcalico/calico/blob/master/manifests/generate.sh#L119-L128 which replaces the versions for our "static" manifests. This way the release process will pick up these changes automatically. This could be used to either replace `VERSION` or `CalicoVersion` from my understanding.